--- ../src-base/minecraft/net/minecraftforge/fml/relauncher/libraries/LibraryManager.java
+++ ../src-work/minecraft/net/minecraftforge/fml/relauncher/libraries/LibraryManager.java
@@ -123,38 +123,6 @@
             return new File(LIBRARY_DIRECTORY_OVERRIDE);
         }
 
-        CodeSource source = ArtifactVersion.class.getProtectionDomain().getCodeSource();
-        if (source == null)
-        {
-            FMLLog.log.error("Unable to determine codesource for {}. Using default libraries directory.", ArtifactVersion.class.getName());
-            return new File(minecraftHome, "libraries");
-        }
-
-        try
-        {
-            File apache = new File(source.getLocation().toURI());
-            if (apache.isFile())
-                apache = apache.getParentFile(); //Get to a directory, this *should* always be the case...
-            apache = apache.getParentFile(); //Skip the version folder. In case we ever update the version, I don't want to edit this code again.
-            String comp = apache.getAbsolutePath().toLowerCase(Locale.ENGLISH).replace('\\', '/');
-            if (!comp.endsWith("/"))
-                comp += '/';
-
-            if (!comp.endsWith("/org/apache/maven/maven-artifact/"))
-            {
-                FMLLog.log.error("Apache Maven library folder was not in the format expected. Using default libraries directory.");
-                FMLLog.log.error("Full: {}", new File(source.getLocation().toURI()));
-                FMLLog.log.error("Trimmed: {}", comp);
-                return new File(minecraftHome, "libraries");
-            }
-            //     maven-artifact  /maven          /apache         /org            /libraries
-            return apache          .getParentFile().getParentFile().getParentFile().getParentFile();
-        }
-        catch (URISyntaxException e)
-        {
-            FMLLog.log.error(FMLLog.log.getMessageFactory().newMessage("Unable to determine file for {}. Using default libraries directory.", ArtifactVersion.class.getName()), e);
-        }
-
         return new File(minecraftHome, "libraries"); //Everything else failed, return the default.
     }
 
